src/vstart.sh

```diff
@@ -280,6 +281,7 @@ options:
 	--osds-per-host: populate crush_location as each host holds the specified number of osds if set
 	--require-osd-and-client-version: if supplied, do set-require-min-compat-client and require-osd-release to specified value
 	--use-crush-tunables: if supplied, set tunables to specified value
+	--use-same-port-rgws: run multiple rgws on same port.
```

Comment:
is this actually for vstart? why would one run multiple rgws if they didn't want the abillity to address each separately?

Reply:
> is this actually useful for vstart? why would one run multiple rgws if they didn't want the abillity to address each separately?

i just thought if someone wants to play around this before trying it out
have kept it default off, but if you want i can remove it from vstart

Reply:
i don't think it's worth the complexity there